src/hooks/useListing.js

```diff
@@ -0,0 +1,66 @@
+import {useInfiniteQuery, useQuery} from 'react-query';
+import axios from 'axios';
+import {apiEndpoint, marketContractAddress, zoombiesContractAddress} from "../constants";
+import {getTokenSymbol} from "../utils/auction";
+
+export const getAuctionListings = async (marketContract, zoombiesContract, filters, sorting, page) => {
+  // console.log({filters, sorting})
+
+  const getSortType = () => {
```

Comment:
nit, doesn't seem like this function is useful

Reply:
you mean getSortType?

Reply:
yah cause it's only just returning sorting.field with no other operations. You can just directly do `sortBy: sorting.field || ''`

Reply:
I've fixed this